releasenotes/notes/fix-missing-pulse-lib-c370f5b9393d0df6.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Fixed a bug that results in an error when a user tries to load .calibration
+    data of a gate in :class:`.Target` in a particular situation.
+    This occurs when the backend doesn't support pulse waveform payload, while
+    Qiskit doesn't have corresponding parametric pulse definition.
+    In this situation, Qiskit pulse object cannot be built, resulting in the failure in
+    parsing PulseQobj and the user encounters a meaningless error.
```

Comment:
What is the situation where the error is meaningless? This changes the code so that the case where a calibration schedule was unloadable behaves like no calibration was reported by the backend rather than raising an exception. If the user wanted to use the calibration, the user's code will still have a problem. Maybe there is some code that eagerly handles calibrations and can work okay when they are not available?

Reply:
Qobj is parsed when they are called for the first time (i.e. when a user call `.calibration` of target instruction) and the meaningless error raised when the user called calibration in their program (non-pulse user will never see this). It tries to raise a `QiskitError` but the error message required `.qubits` that the play instruction payload cannot provide, and this resulted in `AttributeError` instead. It was very hard to understand what was happening.

I expect the user implements the code that checks if calibration is available on their side.